roles/kubernetes/node/tasks/loadbalancer/kube-vip.yml

```diff
@@ -6,6 +6,21 @@
     - kube_proxy_mode == 'ipvs' and not kube_proxy_strict_arp
     - kube_vip_arp_enabled
 
+- name: Kube-vip | Check if super-admin.conf exists
+  stat:
+    path: "{{ kube_config_dir }}/super-admin.conf"
```

Comment:
`super-admin.conf` will be created in each cp node?

Reply:
it does't get created on each cp node and only on first node, after first cp node initialize RBAC fully configured and we can use the `admin.conf` on other nodes.

Reply:
The kube-vip pod run on all the cp node, right?  you mean that the kube-vip pod  on the first cp node is use the super-admin.conf, and the kube-vip pod on the other cp nodes is use the `admin.conf`, right?

Reply:
@cyclinder Yes.

NB: But this is only necessary on bootstrap, i.e before `admin.conf` has cluster-admin privileges.

Reply:
Got, thank you for your clarification!

Reply:
/lgtm